pkgs/development/interpreters/python/passthrufun.nix

```
@@ -91,9 +91,9 @@ in rec {
     pythonAtLeast = lib.versionAtLeast pythonVersion;
     pythonOlder = lib.versionOlder pythonVersion;
     inherit hasDistutilsCxxPatch;
-    # TODO: deprecate
-    # Not done immediately because its likely used outside Nixpkgs.
-    pythonForBuild = pythonOnBuildForHost_overridden;
+    pythonForBuild =
+      lib.warn "pythonForBuild is deprecated; please use pythonOnBuildForHost instead"
```

```suggestion
      lib.warnIf (lib.isInOldestRelease 2311) "`pythonForBuild` (from `python*`) has been renamed to `pythonOnBuildForHost`"
```

Same format I used for the `overrideScope'` deprecation in `makeScope`